platform-includes/configuration/data-to-all-spans/unity.mdx

```diff
@@ -0,0 +1,22 @@
+```csharp
+SentrySdk.Init(options =>
```

Comment:
In different frameworks the main funciton name is different (e.g: `UseSentry` on MAUI and ASP.NET Core, `SentryUnity.Init` or some callback that passes the `options` directly on Unity), so we just go with:

```suggestion
options =>
```